contrib/julia-mode.el

```diff
@@ -131,6 +131,23 @@ This function provides equivalent functionality, but makes no efforts to optimis
              "\\\\"))
         (group "'"))))
 
+(defconst julia-hanging-assignment-regexp
+  (concat "^[^#\n]+ "
+          (regexp-opt '("=" "+=" "-=" "*=" "/=" "//=" "\\=" "^=" "÷=" "%=" "|=" "&=" "$=" "<<=" ">>="
+                        ".+=" ".-=" ".*=" "./=" ".//=" ".\\=" ".^=" ".÷=" ".%="))
+          (regexp-opt '(" #" " \n" "#" "\n"))))
+
+(defconst julia-hanging-operator-regexp
+  (concat "^[^#\n]+ "
+          (regexp-opt '("^" ".^" "//" ".//" "*" "/" "%" "&" "\\" ".*" "./" ".%" ".\\"
+                        "<<" ">>" ">>>" ".<<" ".>>" ".>>>"
+                        "+" "-" "|" "$" ".+" ".-"
+                        ":" ".." "|>"
+                        ">" "<" ">=" "<=" "==" "===" "!=" "!==" "<:" "≠" "≤" "≥"
+                        ".>" ".<" ".>=" ".<=" ".==" ".!="
+                        "&&" "||" "?"))
+          (regexp-opt '(" #" " \n" "#" "\n"))))
```

Comment:
Thanks for the explaination, I don't really think the current behavior is particularly important (since other language modes do not do this). You can probably find why it is introduced with [git-blame](https://github.com/JuliaLang/julia/blame/master/contrib/julia-mode.el)